inc/Admin.php

```diff
@@ -234,12 +231,15 @@ public static function assets( $hook ) {
 
 		$screen = get_current_screen();
 
-			// These assets will be loaded in the bluehost app space only
+		// Load assets only in the Bluehost app space
 		if ( false !== stripos( $hook, 'bluehost' ) ) {
-			// only enqueue on bluehost pages
-			if ( false !== strpos( $screen->id, 'bluehost' ) ) {
-				\wp_enqueue_script( 'bluehost-script' );
-				\wp_enqueue_style( 'bluehost-style' );
+			// Ensure we're on the Bluehost admin page before enqueuing scripts
+			if ( isset( $screen->id ) && false !== strpos( $screen->id, 'bluehost' ) ) {
+				// TODO: Replace this with a proper dependency handling mechanism
+				do_action( 'newfold/installer/enqueue_scripts' );
```

Comment:
We should validate that moving this into the bluehost only pages doesn't break plugin installs from the pages we have in the plugins section.

Reply:
@wpscholar I asked @asr0393 to test the installation of solutions on both the BH plugin page as well as the My Plugins and Tools page, and she was seeing the install popup from the installer module.

She found an issue with Ecomdash that is unrelated to this and will be logging a separate bug for it.